### DISCUSSION TITLE
2 Issues in New Project File After Merging Excel Files

### DISCUSSION THREAD

**Discussion by Morle2104** [asreview/asreview#1924] [Q&A] [answered]
Dear all,

To switch from double to single screening, we merged the xlsx ASReview output files in Excel, and dealt with conflicting labels. Now I uploaded an Excel file with the column label_included and created a new project. ASReview recognizes all labels correctly as prior knowledge, **however there are two strange things that happened.**
**Number 1,** in Analytics it gives the number 1.599 for 'irrelevant records since last relevant', but I have no idea where this number comes from. I uploaded the Excel and made a new project and have not started screening again. In the Excel with which I created the new project, there are only columns related to the articles (pubmed id, abstract, title etc.) and the label_included column. However, it is still the structure of the Excel output file of ASReview in the sense that all as relevant labelled articles are at the top and all as irrelevant at the bottom. So, could it potentially use that to give this 'irrelevant records since last relevant' number?

**The second strange thing** is that I uploaded the same Excel sheet a couple of times, and everytime it gives me a different first abstract to screen. Could this be related to the random seed? I chose the query strategy 'maximum'. 

Many thanks in advance!

**Comment by Rensvandeschoot** [comment, marked as answer]
Thank you for reporting these two issues.

The first issue you encounter is described here https://github.com/asreview/asreview/issues/1747 and has been solved in https://github.com/asreview/asreview/pull/1766 (available in the next release!)

The second issue is probably caused by several relevance scores being very close and depending on some random process of the model every time you see a slightly different paper. You might want to read [this paper](https://www.mdpi.com/2076-3417/14/9/3842) explaining more about the process. If you still think something needs to be corrected, feel free to reach out and discuss it further!

**Reply by Morle2104** [reply]
Many thanks for the quick response! This really helps and clarifies it! :-)
